src/logging.cpp

```diff
@@ -15,7 +15,13 @@
 #include <boost/log/expressions.hpp>
 #include <boost/log/sinks.hpp>
 #include <boost/log/sources/severity_logger.hpp>
-#include <display_device/logging.h>
+
+// conditional includes
+#ifdef __ANDROID__
+  #include <android/log.h>
+#else
+  #include <display_device/logging.h>
```

Comment:
Unless there are some compilation issues, I would not touch the `display_device. This function https://github.com/LizardByte/Sunshine/blob/5da5293471272fa643863a0f3cbdb190a9476a4b/src/display_device.cpp#L615 only creates a SettingsManager for Windows at the moment.

This means that every `display_device` function will either do nothing or act as a "pass-trough". Same for logging - nothing should be logged via `setup_libdisplaydevice_logging`, so maybe just keep it as it is?

Reply:
It's from here: https://github.com/LizardByte/libdisplaydevice/blob/master/src/common/include/display_device/logging.h